meteor-app/imports/api/assessments/parts.js

```diff
@@ -2,7 +2,7 @@ import { Mongo } from 'meteor/mongo'
 import SimpleSchema from  'simpl-schema'
 import { RegExId, createdAt, updatedAt } from '/imports/api/schema'
 
-const Parts = new Mongo.Collection('parts')
+const Parts = new Mongo.Collection('partsData')
```

Comment:
This is a bad pattern. collection names should be in step with variable names. Talk to me about where this conflict is coming from